docs/src/pages/quasar-utils/other-utils.md

```diff
@@ -5,10 +5,11 @@ title: Other Utils
 ``` js
 import { openURL } from 'quasar'
 
-openURL('http://...')
+openURL(String url, Boolean openExternal)
 ```
 
 It will take care of the quirks involved when running under Cordova or on a browser, including notifying the user he/she has to acknowledge opening popups.
+`OpenExternal` option able to choose between opening the URL in a new tab or not. This is useful for link like `tel:+1-123-456-789`.
```

Comment:
The `openExternal` boolean enables choosing between opening the URL in a new tab or not. This is useful for a link like `tel:+1-123-456-789` that should ......... .

Reply:
Explain clearly what it should / should not do